src/test/kotlin/com/terraformation/backend/documentproducer/db/VariableImporterTest.kt

```diff
@@ -752,6 +755,25 @@ class VariableImporterTest : DatabaseTest(), RunsAsUser {
           result.errors)
     }
 
+    @Test
+    fun `returns an error if the deliverable configuration is not complete`() {
```

Comment:
If we decide to use explicit position numbers, we should have a test that checks whether we detect duplicate position numbers for the same deliverable.